src/components/jsCdnTag.js

```diff
@@ -19,7 +19,7 @@ export default ({ apm = false }) => {
     });
   }
 
-  const packageName = apm ? "bundle.apm.min.js" : "bundle.min.js";
+  const packageName = tracing ? "bundle.tracing.min.js" : "bundle.min.js";
```

Comment:
I think we need to update `src/platforms/javascript/index.mdx` and `src/wizard/javascript/index.md`, they reference `bundle.apm.min.js`.

Reply:
Also `src/wizard/javascript/react.md`.